docs/installing.md

```diff
@@ -84,26 +64,29 @@ You can now:
 ansible-playbook -i inventory/hosts setup.yml --tags=start
```

Comment:
I wonder if we should add `ensure-matrix-users-created` here to be consistent with what we propose above.

In most cases, `ensure-matrix-users-created` may not be necessary, because people would be importing an existing database which should contain all bridge/bot users, as expected.

However, we don't really know what kind of database dump users are importing and whether or not they have decided to enable some extra services during their initial installation. Perhaps the user enabled some extra bot.

In that case, it would be safer to add `ensure-matrix-users-created` here.

```suggestion
ansible-playbook -i inventory/hosts setup.yml --tags=ensure-matrix-users-created,start
```